stock_move_location/wizard/stock_move_location.py

```diff
@@ -138,6 +138,20 @@ def _prepare_wizard_move_lines(self, quants):
 
     @api.onchange("origin_location_id")
     def _onchange_origin_location_id(self):
+        if self.origin_location_id:
```

Comment:
Why not making the picking type computed in order to get the default value in the same function ?